spec/acceptance/nodesets/amazonlinux-201703-docker.yml

```diff
@@ -0,0 +1,16 @@
+HOSTS:
+  amazonlinux-201703:
+    roles:
+      - agent
+    platform: el-6-x86_64
+    hypervisor: docker
+    image: amazonlinux:2017.03
+    docker_preserve_image: true
```

Comment:
what does this do?

Reply:
Allows container to be re-used between tests, useful for testing changes on local systems.

https://github.com/puppetlabs/beaker-docker/blob/master/docker.md#preserve-docker-image

These YAML may need to be updated to also allow `BEAKER_provision=no` which has options likely needed with beaker 3.x to fully support re-using containers.

https://github.com/puppetlabs/beaker-docker/blob/master/docker.md#reuse-docker-image

I'll update all images as really handy when doing local development to be able to re-run tests.  There is a bug though where executions after first will put the module in /etc/puppet/modules for some reason which is not used by AIO Puppet.  Never got around to opening an issue with Puppetlabs.

Reply:
FYI - reusable docker images works as documented but annoying bug of module being installed to wrong place is still present:

```
centos-7 20:07:02$ echo /etc/puppet/modules
  /etc/puppet/modules

centos-7 executed in 0.12 seconds
Using scp to transfer /home/treydock/git/sensu-puppet to /etc/puppet/modules/sensu
```

Reply:
Could you please report this bug to Puppet.